db/cli/apply_all_migrations.sh

```diff
@@ -0,0 +1,8 @@
+echo 'Applying migrations'
+for f in source/migrations/*.sql;
```

Comment:
This makes me a little nervous - the sort order is fairly consistent, but there are things that can affect it. Also, it's pretty dumb - if the project ever gets 10+ migrations, I believe this will run # 1, then # 10, then # 2... (Or maybe it starts with # 10 and then # 1?)

I believe this would work:

```suggestion
for f in `ls -v source/migrations/*.sql`;
```

Alternatively, you could make a small command-line wrapper around [migration_handler.ts](https://github.com/express-rate-limit/rate-limit-postgresql/blob/61ce69d6f6b2921db674f744cf0ae4bbbd68a6a9/source/util/migration_handler.ts) and execute that instead. (Although, if this is just supposed to be a test of the migration scripts, then I kind of like the bash version better for it's simplicity. You could always drop the tables and do a second test that runs the migrations via node to cover that case.)

Reply:
Thanks, I completely missed this, I switched to leaving the first init script with `1-` as prefix, and moved the rest to the standard of using `unix-time-` as a prefix. I also added a script for generating a new migration for the new prefix logic.

Reply:
Had to switch back to incremental prefixes due to requirements from [postgres-migrations](https://github.com/thomwright/postgres-migrations#file-name)